liblwgeom/bytebuffer.c

```diff
@@ -22,61 +22,53 @@
  *
  **********************************************************************/
 
-
-
 #include "liblwgeom_internal.h"
 #include "bytebuffer.h"
 
 /**
-* Allocate a new bytebuffer_t. Use bytebuffer_destroy to free.
-*/
-bytebuffer_t*
+ * Allocate a new bytebuffer_t. Use bytebuffer_destroy to free.
+ */
+bytebuffer_t *
 bytebuffer_create(void)
 {
-	LWDEBUG(2,"Entered bytebuffer_create");
+	LWDEBUG(2, "Entered bytebuffer_create");
 	return bytebuffer_create_with_size(BYTEBUFFER_STARTSIZE);
 }
 
 /**
-* Allocate a new bytebuffer_t. Use bytebuffer_destroy to free.
-*/
-bytebuffer_t*
+ * Allocate a new bytebuffer_t. Use bytebuffer_destroy to free.
+ */
+bytebuffer_t *
 bytebuffer_create_with_size(size_t size)
 {
-	LWDEBUGF(2,"Entered bytebuffer_create_with_size %d", size);
+	LWDEBUGF(2, "Entered bytebuffer_create_with_size %d", size);
 	bytebuffer_t *s;
 
 	s = lwalloc(sizeof(bytebuffer_t));
-	if ( size < BYTEBUFFER_STATICSIZE )
-	{
+	if (size < BYTEBUFFER_STATICSIZE) {
```

Comment:
The placement of the initial brace isn't following the style (neither Postgis's or postgresql's)

Reply:
what am I missing?
it's what pgindent produces when I launch it from @strk's pull request:
https://github.com/postgis/postgis/pull/226/files#diff-6bc61232f3b565f3dcc84d60ad377e21R95
I see that style in Postgres repo is different.

Reply:
Dunno, but I see this other style in Postgres:
```c
	if (evtHandle == INVALID_HANDLE_VALUE)
	{
		evtHandle = RegisterEventSource(NULL,
										event_source ? event_source : DEFAULT_EVENT_SOURCE);
		if (evtHandle == NULL)
		{
			evtHandle = INVALID_HANDLE_VALUE;
			return;
		}
	}
```